### PR TITLE
Latest Items

### PR DESCRIPTION
**Pull request by finish06**
Added ability to scrape recent item additions to the Emby library.  Could be used to show recent library changes in Home-Assistant.
Let me know of any possible changes, etc, as this is my first time doing this.
Thanks!

### CODE REVIEW

**Comment by mezz64** [comment]
Thanks for the PR!  Looks OK from a quick look.  I'll get a chance to test and hopefully merge this weekend.

**Comment by finish06** [comment]
Requested changes have been made.  Thanks for pointing out all the lint errors.  I ran it by pylint and I think I have them all corrected.

File: pyemby/emby.py [hunk 1]
```diff
@@ -98,3 +98,22 @@ def get_image(self, item_id, style, played=0):
...
         return '{0}/Items/{1}/Images/{2}?api_key={3}&PercentPlayed={4}'.format(
             self.server_url, item_id, style, self.api_key, played)
+
+    def get_latest_items(self, session, limit=3, is_played='false',
+                         include_item_types='episode'):
+        """Return XX most recent movie or episode additions to library"""
+        query = 'IncludeItemTypes={0}&Limit={1}&IsPlayed={2}'
+        query = query.format(include_item_types, limit, is_played)
+        try:
+            response = self.emby_request.get(self.server_url +
```
Comment (mezz64) [inline pyemby/emby.py]: Split this up like this to make it more readable:
```python
response = self.emby_request.get(
                self.server_url + '/Users/{0}/Items/Latest?{1}'.
                format(session['UserId'], query))
```

File: pyemby/emby.py [hunk 2]
```diff
@@ -98,3 +98,22 @@ def get_image(self, item_id, style, played=0):
...
+                                             '/Users/{0}/Items/Latest?{1}'.
+                                             format(session['UserId'],
+                                                    query))
+        except KeyError as err:
+            _LOGGER.error('Requests error getting key: %s', err)
+            return
+        except requests.exceptions.RequestException as err:
```
Comment (mezz64) [inline pyemby/emby.py]: You can combine this with the KeyError so there is only one exception line:
```python
except (KeyError, requests.exceptions.RequestException) as err:
```
Then just make the message a little less specific.

**Comment by mezz64** [comment]
Once you get those changes taken care of it looks good to merge.

File: pyemby/emby.py (hunk 1 above)
Comment (finish06) [inline pyemby/emby.py]: Pylint recommends I do it as follows:
`
        try:
            response = self.emby_request.get(
                self.server_url + '/Users/{0}/Items/Latest?{1}'.
                format(session['UserId'], query))
`

What would you prefer?

Comment (finish06) [inline pyemby/emby.py]: Pylint recommends the following:
```
        try:
            response = self.emby_request.get(
                self.server_url + '/Users/{0}/Items/Latest?{1}'.
                format(session['UserId'], query))
```

What would you prefer?

**Pull request merged**